docs/changelog/88270.yaml

```diff
@@ -0,0 +1,5 @@
+pr: 88270
+summary: Updatable API keys - REST API spec and tests
+area: Authentication
```

Comment:
I am curious why you picked `Authentication`. I'd go with the generic `Security`. It's not great either. I don't have a strong opinion, but would like to know your thoughts.

Reply:
I really wasn't sure here. I went with authentication because API keys generally are related to authentication but it's a bit of a stretch for this PR. I think `Security` might make more sense.